nw-firestore-schema.ts

```diff
@@ -2,13 +2,16 @@ interface Sponsor {
   name: "apple";
   url: "apple.com";
   imgURL: "link-to-image";
+  lastmod: Date().toLocaleString(),
+  teir: "platinum"
```

Comment:
Typo in teir, also could you make this a type, so its easier to understand what the options are?

aka
```typescript
type Tier = "gold" | "silver" | "platinum"
````

and then in the sponsors interface:

```typescript
interface Sponsor {
tier: Tier = "gold";
}
```